kubernetes_tests/test_kubernetes_pod_operator.py

```diff
@@ -903,7 +904,6 @@ def test_on_kill(self,
         self.assertEqual(pod.status.phase, "Running")
         k.on_kill()
         with self.assertRaises(ApiException):
-            # pod should be deleted
-            client.read_namespaced_pod(name=name, namespace=namespace)
+            pod = client.read_namespaced_pod(name=name, namespace=namespace)
```

Comment:
Can we add a test that verifies that `termination_grace_period` is passed correctly i.e. mocking `delete_namespaced_pod` and just doing assert_called_with

Reply:
@kaxil the termination_grace_period is being passed because without passing it this test would fail. read_namespaced_pod only returns an error when the pod is fully deleted and this pod would take 20 seconds to delete without a termination_grace_period.

Reply:
aah, fair enough